.vscode/tasks.json

```diff
@@ -1,16 +1,21 @@
 {
```

Comment:
It so happened that I opened the project in VS Code. I received a message that the format of this file is outdated and it can be migrated.